projects/core/src/lib/download-vector-layer/download-vector-layer.component.html

```diff
@@ -0,0 +1,71 @@
+
+<form [formGroup]="form">
+    <mat-dialog-content>
+        <mat-card>
+            <mat-card-header>
+                <mat-card-title>Area</mat-card-title>
+                <mat-card-subtitle>Specify a rectangular region for the download</mat-card-subtitle>
+            </mat-card-header>
+
+            <mat-card-content>
+                <div fxLayout="row">
+                    <mat-form-field appearance="fill">
+                        <mat-label>min X</mat-label>
+                        <input matInput type="number" formControlName="bboxMinX" />
+                    </mat-form-field>
+
+                    <mat-form-field appearance="fill">
+                        <mat-label>max X</mat-label>
+                        <input matInput type="number" formControlName="bboxMaxX" />
+                    </mat-form-field>
+
+                    <mat-form-field appearance="fill">
+                        <mat-label>min Y</mat-label>
+                        <input matInput type="number" formControlName="bboxMinY" />
+                    </mat-form-field>
+
+                    <mat-form-field appearance="fill">
+                        <mat-label>max Y</mat-label>
+                        <input matInput type="number" formControlName="bboxMaxY" />
+                    </mat-form-field>
+                </div>
+
+                <button mat-raised-button color="accent" (click)="selectBox()" [disabled]="isSelectingBox">Select region on the map</button>
+            </mat-card-content>
+        </mat-card>
+
+        <mat-card>
+            <mat-card-header>
+                <mat-card-title>Time</mat-card-title>
+                <mat-card-subtitle>Specify a time selection for the download</mat-card-subtitle>
+            </mat-card-header>
+
+            <mat-card-content>
+                <geoengine-time-interval-input formControlName="timeInterval"></geoengine-time-interval-input>
+            </mat-card-content>
+        </mat-card>
+
+        <mat-card>
+            <mat-card-header>
+                <mat-card-title>Resolution</mat-card-title>
+                <mat-card-subtitle>Specify a resolution for the download</mat-card-subtitle>
+            </mat-card-header>
+
+            <mat-card-content>
+                <mat-form-field appearance="fill">
+                    <mat-label>Resolution</mat-label>
+                    <input matInput type="number" formControlName="resolution" />
+                    <mat-hint
+                        class="error"
+                        *ngIf="form.controls.resolution.errors?.required || form.controls.resolution.errors?.valueNotLarger"
+                    >
+                        The resolution must be a positive number.
+                    </mat-hint>
+                </mat-form-field>
+            </mat-card-content>
+        </mat-card>
+
+        <button type="submit" mat-raised-button color="primary" [disabled]="form.invalid" (click)="download()">Download</button>
```

Comment:
![grafik](https://github.com/geo-engine/geoengine-ui/assets/4947061/1a5f4e16-c0ea-425b-8348-96df1a291383)

In our dialogs, we have a scrolling mechanism but show the action button always on the bottom. Moreover, it is on the right.

![grafik](https://github.com/geo-engine/geoengine-ui/assets/4947061/33fcd8dc-258e-4b30-84b3-22bfa5642823)


Could this dialog be similar to the other dialog? For instance, we could re-use the layout component.

Reply:
https://github.com/geo-engine/geoengine-ui/pull/722/commits/0f9ea020f7c11e0c8381eea2deddfdd272dea3f3